scalafix-cli/src/main/scala/scalafix/internal/cli/ScalafixOptions.scala

```diff
@@ -144,7 +144,23 @@ case class ScalafixOptions(
     @HelpMessage("""Print out sbt completion parser to argument.""".stripMargin)
     @ValueDescription("scalafix-sbt/src/main/scala/Completions.scala")
     @Hidden
-    sbt: Option[String] = None
+    sbt: Option[String] = None,
+    @HelpMessage(
+      "If set, only apply scalafix to added and edited files in git diff against master.")
+    diff: Boolean = false,
+    @HelpMessage(
+      "If set, only apply scalafix to added and edited files in git diff against a provided branch.")
+    diffBranch: Option[String] = None,
+    @HelpMessage(
+      """|If set, only scalafix to added and edited files in a git diff provided in the stdin
+         |
+         |scalafix --diff
```

Comment:
Can we move these examples to the list of other examples in Cli.helpMessage?

Reply:
Sure, in user doc?

Reply:
See `Cli.helpMessage` "examples" section

Reply:
Can we remove this message here now that's it's explained with other examples.